README.md

```diff
@@ -9,6 +9,42 @@ With ease:
 - publish page using GitHub-Pages,
 - mirror changes to a separate repository.
 
+## Requirements and Prerequisites
+
+To ensure your GitHub Actions workflows function correctly, it's important to configure the `GITHUB_TOKEN` with the appropriate access rights for each repository. 
+
+Follow these steps to set up the necessary permissions:
+1. Navigate to your repository on GitHub.
+2. Click on `Settings` located in the repository toolbar.
+3. In the left sidebar, click on `Actions`.
+4. Under the `Actions` settings, find and click on `General`.
+5. Scroll down to the `Workflow permissions` section.
+6. You will see the default permission setting for the `GITHUB_TOKEN`. Click on the option for `Read and write permissions`.
+7. With this setting, your workflow will have the ability to read the contents of the repository and push back changes, which is required for using this GitHub Action.
+
+Make sure to save your changes before exiting the settings page.
+
+> \[!NOTE]
+>
+> Granting `Read and write permissions` allows workflows to modify your repository, which can include adding or updating files and code. Always ensure that you trust the workflows you enable with these permissions.
+![General Settings](docs/images/Github_Initial_Git_Push_Settings_General.png)
```

Comment:
Can we add a new line here to prevent the image appearing below the blockquote.

```suggestion

![General Settings](docs/images/Github_Initial_Git_Push_Settings_General.png)
```

Reply:
Done